CDTDatastore/Attachments/CDTDatastore+Attachments.m

```diff
@@ -230,26 +231,19 @@ - (BOOL)addAttachment:(NSDictionary *)attachmentData
     NSDictionary *params;
 
     // delete any existing entry for this file and sequence combo
-    params = @{ @"filename" : filename, @"sequence" : @(sequence) };
-    success = [db executeUpdate:[SQL_DELETE_ATTACHMENT_ROW copy] withParameterDictionary:params];
+    success = [db executeUpdate:[SQL_DELETE_ATTACHMENT_ROW copy]
```

Comment:
I gave this a bit of thought, and decided it was better to roll this code back, and inspect the error with `FMDB.lastError`. I did that in 9b6f791